setup.cfg

```diff
@@ -0,0 +1,15 @@
+[flake8]
+show-source = True
+ignore = E203, E731, W503
+max-line-length = 88
```

Comment:
here 88 vs 80 in pyproject.toml

As said in the other project - is there a reason for this or is this accidental?

Reply:
There is a reason.
Black is not very conservative with the line length and allows sometimes for more. 
It's recommended do add about 10%.
For Morepath I even needed more.

Reply:
Thanks for this information, @henri-hulski ! I learned something new today!